### PR TITLE
Updated to ES6 + Mixins + Nunjucks

### PR DESCRIPTION
**Pull request by joaqtor**

### CODE REVIEW

**Comment by pepoviola** [comment]
ping @dgbeck, this looks good to me.
Thx!

**Comment by dgbeck** [comment]
@joaqtor i'm seeing 323 files changes in this PR

image

are we missing a `.gitignore`?

**Comment by joaqtor** [comment]
That's because `node_modules` was commited before. The 323 changes is `node_modules` being removed.

File: src/moduiCalendarPane.nunj [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                <a class="nav-btn next-btn {{ "disabled" if isNextBtnDisabled }}" data-nav="next">
+                    &#9658;
+                </a>
+                <a class="nav-btn prev-btn {{ "disabled" if isPrevBtnDisabled }}" data-nav="prev">
+                    &#9668;
+                </a>
+            {% endif %}
+            <h6>
+                {% if displayYearBeforeMonth %}
+                    {{month.year}} {{month.label}}
```
Comment (dgbeck) [inline src/moduiCalendarPane.nunj]: let's put spaces here (and everywhere) in between curly braces and the variable name..

```
{{ month.year }} {{ month.label }}
```

**Comment by joaqtor** [comment]
@dgbeck resolved all comments!

**Comment by dgbeck** [comment]
🎸 rock on!

**Comment by joaqtor** [comment]
4893d669907a21da

**Pull request merged**